index.md

```diff
@@ -81,7 +81,7 @@ title: .NET Documentation
                                                 </div>
                                             </div>
                                             <div class="cardText">
-                                                <h3><a href="/dotnet/articles/welcome">.NET Platform Guide</a></h3>
+                                                <h3><a href="/dotnet/articles/standard">.NET Platform Guide</a></h3>
```

Comment:
Were we terminating the ones that will resolve to an *index.md* with a trailing slash, right?

*/dotnet/articles/standard/*